cmd/config.go

```diff
@@ -63,6 +67,16 @@ func (d *DBConfig) URL() (string, error) {
 	return d.DBConnect, nil
 }
 
+// GetMaxOpenConns defaults MaxOpenConns to the value of MaxDBConns if
+// MaxDBConns was specified in the config json. This method can be
+// removed once MaxDBConns has been removed from test/config.
```

Comment:
Has been removed from test/config _and_ all prod and staging configs.

Reply:
Fixed and added a TODO linked to a PR